code/modules/projectiles/guns/energy.dm

```diff
@@ -150,7 +150,7 @@
 	ammo = /datum/ammo/energy/lasgun/M43
```

Comment:
missed a m43 reference here?

Reply:
nah, it shoots the current lasgun bullet/bullets. I feel that copying the current lasgun bullets just to rename the typepath is just code duplication, and another thing to keep in mind when balancing.

Reply:
Spooky is right, this reference needs to be renamed unless you can tell me of another weapon that makes use of it.

Reply:
the m43 lasgun uses it. which is technically admin spawn only right now.